app/src/main/java/org/xwiki/android/sync/auth/RegistrationActivity.kt

```diff
@@ -0,0 +1,20 @@
+package org.xwiki.android.sync.auth
+
+import android.os.Bundle
+import android.webkit.WebView
+import androidx.appcompat.app.AppCompatActivity
+import org.xwiki.android.sync.R
+import org.xwiki.android.sync.utils.openLinkInWebView
+
+class RegistrationActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_registration)
+        val webView = findViewById(R.id.webview) as WebView
+        val url = intent.getStringExtra("url")
+        val intent = openLinkInWebView(
+                url, webView
```

Comment:
Usually, it is better to make ALL arguments on a new lines or all in one line. For example:

someCall(argument)

OR

someCall(
    arg1,
    arg2
)

Reply:
Fixed this too.